stablehlo/conversions/linalg/transforms/StablehloLegalizeToLinalg.cpp

```diff
@@ -842,8 +842,7 @@ struct TransposeOpToTransposeConverter final
 
     // TODO(#2216) Cleanup Attribute -> DenseArrayAttr
```

Comment:
Looks like this comment can be removed too since we are reverting the change back to regular arrays.